uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -0,0 +1,141 @@
+//contains data for the mock api calls for entitlement
```

Comment:
Self explanatory. Either you have something specific to comment about (notes, gotchas, anything of the sort), or you can remove this